oio/cli/shell.py

```diff
@@ -43,6 +43,13 @@ def configure_logging(self):
         else:
             requests_log.setLevel(logging.ERROR)
 
+        urllib3_log = logging.getLogger('urllib3')
+
+        if self.options.debug:
+            urllib3_log.setLevel(logging.DEBUG)
+        else:
+            urllib3_log.setLevel(logging.ERROR)
```

Comment:
Is `INFO` too verbose? By the way, if you get rid of requests, you can discard the `'requests'` logger.